### PR TITLE
Add contractors table with name, type and contract_number columns

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Create contractors table with name, type and contract_number columns

## Setup
`make db_dev_migrate`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/169690980) for this change

### CODE REVIEW

File: pkg/models/contractors.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+// This method is not required and may be deleted.
+func (c *Contractor) ValidateCreate(tx *pop.Connection) (*validate.Errors, error) {
+	return validate.NewErrors(), nil
+}
+
+// ValidateUpdate gets run every time you call "pop.ValidateAndUpdate" method.
+// This method is not required and may be deleted.
+func (c *Contractor) ValidateUpdate(tx *pop.Connection) (*validate.Errors, error) {
+	return validate.NewErrors(), nil
+}
```
Comment (LeDeep) [inline pkg/models/contractors.go]: would be good to have a test file for this. You can see some examples in some of the other model implementations.

File: migrations/20191114203559_create-contractors-table.up.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE contractor
+(
+	id UUID PRIMARY KEY,
+    created_at timestamp WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at timestamp WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    name text NOT NULL,
+    contract_number text UNIQUE NOT NULL,
```
Comment (LeDeep) [inline migrations/20191114203559_create-contractors-table.up.sql]: i think the `text` datatype is deprecated. You can use `varchar` and set the size yourself. Something like `varchar(80)` would work.

**Comment by sarboc** [comment]
@Ronolibert Where does this fall into the new db schema? If we have move transit orders and nts transit orders in separate tables, then I'm curious why/where we need to reference contractors.

**Comment by Ronolibert** [comment]
@sarboc I think its because contractors can change. Each contract number is associated to a specific contractor. If they miss their performance scores then they can be replaced

**Pull request merged**